samples/winapi_filefind/main.c

```diff
@@ -38,24 +40,29 @@ int main(void)
         } else {
             debugPrint("File     : ");
         }
-
         debugPrint("%s\n", findFileData.cFileName);
     } while (FindNextFile(hFind, &findFileData) != 0);
 
     debugPrint("\n");
 
-    DWORD error = GetLastError();
-    if (error == ERROR_NO_MORE_FILES) {
+    findFileError = GetLastError();
+    if (findFileError == ERROR_NO_MORE_FILES) {
         debugPrint("Done!\n");
     } else {
-        debugPrint("error: %x\n", error);
+        debugPrint("error: %x\n", findFileError);
     }
 
     FindClose(hFind);
 
+cleanup:
+    ret = nxUnmountDrive('C');
+    // If there was an error while unmounting
+    if (!ret) {
+        DWORD unmountError = GetLastError();
+        debugPrint("Couldn't unmount C: drive! Error code: %x", unmountError);
+    }
+sleepForever:
     while (1) {
         Sleep(2000);
     }
-
-    return 0;
```

Comment:
Why the removal? `main` returns an int, so there should be a return statement, even if it's never reached.